docs/getting-started.md

```diff
@@ -33,7 +33,6 @@ Ensure that you have access to the required team resources. If you encounter a p
 DUOS and [Terra](https://terra.bio/) use [Sam](https://github.com/broadinstitute/sam) to abstract identity and access management. To gain access to these services, first create a non-Broad email address through Gmail. This email address will specifically be used for development purposes in our non-prod environments. Next, to register as a new user, click the `Sign in with Google` button in each of the environments with the newly created email address and follow the prompts:
```

Comment:
As a follow-up, we can think of a single Data Team `getting-started.md` in the future. My goal for the document is that a developer has their environment set up so that they can start developing, which is upstream from running the service locally.